tests/system/python/conftest.py

```diff
@@ -716,6 +716,108 @@ def _read_data_from_pi_web_api(host, admin, password, pi_database, af_hierarchy_
     return _read_data_from_pi_web_api
 
 
+@pytest.fixture
+def read_data_from_pi_asset_server():
```

Comment:
We did not already have a fixture to read data using PI Web API?!

Reply:
Yes, we already have a fixture to read data from the PI Web API, but it lacks the flexibility to read data from different AF hierarchies. It only accesses a specific AF structure and cannot identify specific assets or data points if multiple FogLAMPs are sending data to the same AF location.